plugins/woocommerce/src/Internal/Admin/Orders/ListTable.php

```diff
@@ -1352,7 +1352,7 @@ private function do_bulk_action_remove_personal_data( $order_ids ): int {
 			}
 
 			do_action( 'woocommerce_remove_order_personal_data', $order ); // phpcs:ignore WooCommerce.Commenting.CommentHooks.MissingHookComment
-			$changed++;
+			++$changed;
```

Comment:
hey @senadir, I'm not quite following these changes, what am I missing?

Since the var is not being assigned, only returned after, doing pre-increment shouldn't make a difference right?
[Quick example on how it would make a difference](https://onlinephp.io/c/2b8a0)

Reply:
Linter did it.